components/actionbutton/stories/actionbutton.stories.js

```diff
@@ -1,11 +1,21 @@
 import { default as IconStories } from "@spectrum-css/icon/stories/icon.stories.js";
+import { Sizes } from "@spectrum-css/preview/decorators";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
 import { isActive, isDisabled, isEmphasized, isFocused, isHovered, isQuiet, isSelected, size, staticColor } from "@spectrum-css/preview/types";
 import pkgJson from "../package.json";
-import { ActionButtonGroup, ActionButtons } from "./actionbutton.test.js";
+import { ActionButtonGroup } from "./actionbutton.test.js";
+import { ActionButtonsWithIconOptions, IconOnlyOption, TreatmentTemplate } from "./template.js";
 
 /**
  * The action button component represents an action a user can take.
+ * 
+ * ## Usage notes  
+ * 
+ * For Action Buttons that only contain an icon with no label, do not include the element with the `.spectrum-ActionButton-label` class in the markup. If an icon and a label are both used, ensure that the element with the `.spectrum-ActionButton-label` class comes after the `.spectrum-Icon` element.
+ * 
+ * If the hold icon is used, ensure that the element with the `.spectrum-ActionButton-hold` class comes before the `.spectrum-Icon` element.
+ * 
+ * When using `.spectrum-ActionButton--staticWhite` or `.spectrum-ActionButton--staticblack`, use the `--mod-actionbutton-content-color-default` custom property to set the text color when selected.  
```

Comment:
Could you capitalize this please? `.spectrum-ActionButton--staticeBlack`. It took me a hot second to find these styles, but [static colors are capitalized](https://github.com/adobe/spectrum-css/blob/5f523698503e5683f5a769b0134e092238f7e1b0/components/actionbutton/themes/spectrum.css#L55) in their modifier class names.